### PR TITLE
Batch merge 2.9.0

### PR DESCRIPTION
**Pull request by carlescufi**

### CODE REVIEW

**Comment by carlescufi** [comment]
Replaces https://github.com/nrfconnect/sdk-nrf/pull/19266

File: tests/benchmarks/current_consumption/system_off/testcase.yaml [hunk 1]
```diff
@@ -48,50 +50,39 @@ tests:
...
       fixture: ppk_power_measure
       pytest_root:
         - "${CUSTOM_ROOT_TEST_DIR}/test_measure_power_consumption.py::test_measure_and_data_dump_power_consumption_systemoff"
     timeout: 90
-  benchmarks.current_consumption.sample.boards.nrf.system_off.nrf_retained:
+  benchmarks.current_consumption.sample.boards.nrf.system_off.retained_mem:
     platform_allow:
       - nrf52840dk/nrf52840
       - nrf52dk/nrf52832
-      - nrf5340dk/nrf5340/cpuapp
```
Comment (nordic-segl) [inline tests/benchmarks/current_consumption/system_off/testcase.yaml:71]: nrf5340dk was removed from the platform_allow list. Is this intentional?

Comment (carlescufi) [reply tests/benchmarks/current_consumption/system_off/testcase.yaml:71]: I have no idea, this is cherry-picked from other PRs. Please ask the relevant contributors.

Comment (nordic-segl) [reply tests/benchmarks/current_consumption/system_off/testcase.yaml:71]: https://github.com/nrfconnect/sdk-nrf/pull/19140/files#r1869874817

**Review by bama-nordic — APPROVED** [review]

**Review by jukkar — APPROVED** [review]

**Review by shanthanordic — APPROVED** [review]
Approvings

**Review by D-Triveni — APPROVED** [review]

**Review by sachinthegreen — APPROVED** [review]

**Review by nordic-babu — APPROVED** [review]

**Pull request merged**